src/Take.Elephant.Sql/StorageBase.cs

```diff
@@ -206,6 +206,12 @@ protected async Task<DbConnection> GetConnectionAsync(CancellationToken cancella
         protected CancellationTokenSource CreateCancellationTokenSource()
             => new CancellationTokenSource(DatabaseDriver.Timeout);
 
+        protected CancellationTokenSource CreateCancellationTokenSource(CancellationToken cancellationToken)
+        {
+            var dataBaseTimeOutTokenSource = new CancellationTokenSource(DatabaseDriver.Timeout);
```

Comment:
This method will leak the `dataBaseTimeOutTokenSource`, which implements `IDisposable` and won't be disposed :(